includes/class-wp-job-manager-post-types.php

```diff
@@ -1373,6 +1449,40 @@ public function sitemaps_maybe_hide_filled( $query_args, $post_type ) {
 		return $query_args;
 	}
 
+	/**
+	 * Hide expired job listings in WP core sitemaps when the `Hide expired positions` setting
+	 * is enabled.
+	 *
+	 * @access private
+	 * @since 1.39.0
+	 *
+	 * @param array  $query_args Array of WP_Query arguments.
+	 * @param string $post_type  Post type name.
+	 *
+	 * @return array
+	 */
+	public function sitemaps_maybe_hide_expired( Array $query_args, String $post_type ): array {
```

Comment:
Not sure where is this method being used. Is this some leftover code?

Reply:
Ah, yes, I realised that expired posts are already excluded from sitemaps, so I will remove this.